### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by wQuelS**

### CODE REVIEW

File: cinema/views.py [hunk 1]
```diff
@@ -166,6 +189,22 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type=str,
```
Comment (ketstap162) [inline cinema/views.py:192]: Use type=date

File: cinema/models.py [hunk 2]
```diff
@@ -50,8 +50,8 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
-    genres = models.ManyToManyField(Genre)
-    actors = models.ManyToManyField(Actor)
+    genres = models.ManyToManyField(Genre, blank=True)
+    actors = models.ManyToManyField(Actor, blank=True)
```
Comment (ketstap162) [inline cinema/models.py:54]: Add related name

**Review by ketstap162 — CHANGES_REQUESTED** [review]

**Review by ketstap162 — APPROVED** [review]
Good job!

**Review by Nattalli — CHANGES_REQUESTED** [review]
if you don't remove the authentication classes in manage user - then the detailed page about the user will not work. Check it!

**Pull request closed without merging**